### PR TITLE
Updates to author images displaying

### PR DESCRIPTION
**Pull request by joshdarby**
## Changes

This pull request makes the following changes:

- Created new `borderzine_has_avatar` function, copied from https://github.com/INN/largo/blob/f50df16e9e06fb9402802e2f1ce76ce058d26d76/inc/avatars.php#L34-L54 , to include a check to see if `userphoto_the_author_thumbnail()` exists from the [User Photo](https://wordpress.org/plugins/user-photo/) plugin.
- Updates `borderzine_byline` class to use new `borderzine_has_avatar` function.
- Updates `content-single.php` to use `borderzine_byline` class.
- Updates `author-bio-description.php` partial to use new `borderzine_has_avatar` class.

## Why

For #21 / author images not displaying now that we've figured out how to get them on the staging site.

## Testing/Questions

Features that this PR affects:

- Author images
- Author bylines

Questions that need to be answered before merging:

- [ ] Are there any more cases of `largo_has_avatar` we should swap to use `borderzine_has_avatar`?

Steps to test this PR:

1. Install the [User Photo](https://wordpress.org/plugins/user-photo/) plugin and verify authors now have images in their profile
2. View an author byline on a post and verify their picture shows
3. View an author bio widget and verify their picture shows

### CODE REVIEW

**Comment by benlk** [comment]
Worked for me once I downloaded `wp-content/uploads/userphoto`.

Are we concerned about small or non-square avatars?

Screen Shot 2019-09-25 at 16 10 56
Screen Shot 2019-09-25 at 16 11 10
Screen Shot 2019-09-25 at 16 11 25

File: wp-content/themes/borderzine/functions.php [hunk 1]
```diff
@@ -75,4 +75,29 @@ function borderzine_do_sidebar( $args ) {
...
+ * @since 0.1
+ */
+function borderzine_has_avatar( $email ) {
+	$user = get_user_by( 'email', $email );
+	if ( ! empty( $user ) ) {
+		$result = largo_get_user_avatar_id( $user->ID );
+		if ( ! empty ( $result ) ) {
+			return true;
+		// this checks if the user has a photo placed by the User Photo plugin
+		} else if( userphoto_the_author_thumbnail() ){
```
[the remark this answers is not in the document]
Comment (joshdarby) [reply wp-content/themes/borderzine/functions.php]: Good catch!

[the remark this answers is not in the document]
Comment (joshdarby) [reply wp-content/themes/borderzine/functions.php]: Fixed in https://github.com/INN/umbrella-borderzine/pull/38/commits/5bc2b1a366612fdb5dff3ca515751f970a271c51.

**Comment by joshdarby** [comment]
> Are we concerned about small or non-square avatars?

@kaylima (on slack) says we should standardize them and put them on the left side of the bio instead of on top.

**Comment by joshdarby** [comment]
@kaylima Is this close to what you were hoping for?

Screen Shot 2019-09-25 at 6 13 48 PM

**Comment by benlk** [comment]
Looks good, but I think we'll run into lots of "The avatar is too small" problems, which can be resolved by having them upload new avatars.

Screen Shot 2019-09-25 at 19 25 26
Screen Shot 2019-09-25 at 19 25 30
Screen Shot 2019-09-25 at 19 25 36

**Pull request merged**